src/caret_analyze/plot/plot_facade.py

```diff
@@ -273,3 +303,81 @@ def create_message_flow_plot(
             target_path, visualize_lib, granularity, treat_drop_as_delay, lstrip_s, rstrip_s
         )
         return plot
+
+    @staticmethod
+    def create_frequency_histogram_plot(
+        *target_objects: HistTypes
+    ) -> PlotBase:
+        """
+        Get frequency histogram plot instance.
+
+        Parameters
+        ----------
+        target_objects : Collection[HistTypes]
+            HistTypes = CallbackBase | Communication
+            Instances that are the sources of the plotting.
+            This also accepts multiple inputs by unpacking.
+
+        Returns
+        -------
+        PlotBase
+
+        """
+        visualize_lib = VisualizeLibFactory.create_instance()
+        plot = HistogramPlotFactory.create_instance(
+            parse_collection_or_unpack_for_hist(target_objects),  # type: ignore
```

Comment:
Please do not use `# type: ignore`.

The same applies to `create_latency_histogram_plot`, and `create_period_histogram_plot`.